tests/integration-tests/tests/storage/test_fsx_lustre/test_fsx_lustre/pcluster.config.ini

```diff
@@ -38,3 +38,4 @@ deployment_type = {{ deployment_type }}
 {% if per_unit_storage_throughput %}
 per_unit_storage_throughput = {{ per_unit_storage_throughput }}
 {% endif %}
+auto_import_policy = NEW_CHANGED
```

Comment:
Do you think it's worth testing `NEW` and `NONE` as well? There are already three test cases, so it wouldn't cost much extra time to add a third parameter and pass it into this config, and then add logic to `_test_auto_import` that asserts different conditions based on which policy was used.